views/my_packages.register.dt

```diff
@@ -16,7 +16,7 @@ block body
 		h1 Add new package
 			p.light
 				a.blind(href="#{req.rootDir}publish") Learn more
-				| about publishing.
+				|  about publishing.
```

Comment:
Unrelated

Reply:
this comes from the other PR, we should get that one merged before doing this PR. I just put it here for now for information and how the implementation will basically look.